pkg/operator/operator.go

```diff
@@ -163,7 +163,7 @@ func (o *Options) defaultAndValidate(_ logr.Logger) error {
 	return nil
 }
 
-// NewScheme creates a new Kubernetes runtime.Scheme for the GMP Operator
+// NewScheme creates a new Kubernetes runtime.Scheme for the GMP Operator.
```

Comment:
should be commas? or on make comment a single line.

Reply:
Not sure what you mean. This is the `godot` linter ensuring there is a period at the end of the sentence.

Reply:
sorry, this ended up in the wrong line. It should have been in line 210.